x/feegrant/client/rest/grpc_query_test.go

```diff
@@ -1,210 +0,0 @@
-package rest_test
```

Comment:
I think we shouldn't delete this, This file only contains grpc gateway tests.

cc: @amaurym

Reply:
usually this folder is for legacy REST
I don't think we should keep using it for grpc gateway
I have basically moved those tests to `keeper/grpc_query_test.go` as discussed during our audit